server/data/contactsApiClient.ts

```diff
@@ -43,4 +45,25 @@ export default class ContactsApiClient extends RestClient {
       user,
     )
   }
+
+  async searchContact(
+    contactSearchRequest: ContactSearchRequest,
+    user: Express.User,
+    pagination?: Pageable,
+  ): Promise<Contact> {
+    const paginationParameters = pagination ?? { page: 0, size: config.apis.prisonerSearchApi.pageSize || 10 }
```

Comment:
I think this variable `config.apis.prisonerSearchApi.pageSize` is set to 20.  Shoulduse a different variable for contact list page size or just make it a constant

Reply:
`pageSize` added to `config.apis.contactsApi`